src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -25,6 +25,7 @@ class Lambda {
           sendFastlyPurgeRequestAndAmpPingRequest(event.payloadId, Hard)
         case (ItemType.Content, EventType.Update) =>
           sendFastlyPurgeRequest(event.payloadId, Soft)
+          sendFastlyPurgeRequest(s"${event.payloadId}.json", Soft, config.fastlyApiNextgenServiceId)
```

Comment:
As discussed, just a quick check -- is there a way of documenting whatever dependency this string represents at the point at which it is defined?

Reply:
Good shout: https://github.com/guardian/frontend/pull/21480/files